contracts/GovernorAlpha.sol

```diff
@@ -302,6 +302,6 @@ interface TimelockInterface {
     function executeTransaction(address target, uint value, string calldata signature, bytes calldata data, uint eta) external payable returns (bytes memory);
 }
 
-interface UniInterface {
+interface VexInterface {
```

Comment:
Do we want to move `interface` to the top? Having it at the bottom seems confusing lol. Same would go for `TimelockInterface`

Reply:
okay fixed